odl/contrib/fom/supervised.py

```diff
@@ -130,7 +130,8 @@ def mean_absolute_error(data, ground_truth, mask=None, normalized=False):
     return fom
 
 
-def mean_value_difference(data, ground_truth, mask=None, normalized=False):
+def mean_value_difference(data, ground_truth, mask=None, normalized=False,
+                          force_lower_is_better=False):
     """Return difference in mean value between ``data`` and ``ground_truth``.
```

Comment:
Can you change this to a raw string with r"""string""" and remove the double backslashes in the docstring? Otherwise, I think that you should change \mathrm to \\mathrm.